x/evm/keeper/msg_assigner.go

```diff
@@ -56,18 +74,29 @@ func getValidatorsExecutionTime(validators []valsettypes.Validator) map[string]f
 	return validatorsSuccessRate
 }
 
+func getValidatorsFeatureSet(validators []valsettypes.Validator) map[string]float64 {
+	validatorsFeatureSet := make(map[string]float64, len(validators))
+	for _, validator := range validators {
+		// Placeholder until we start tracking validator executionTime
```

Comment:
Very minute.  Copy pasta "executionTime" in the comment.

Reply:
Fixed!